.prettierrc.yml

```diff
@@ -0,0 +1,14 @@
+trailingComma: all
+printWidth: 80
+singleQuote: true
+arrowParens: avoid
```

Comment:
I think we should remove this and allow it to be defaulted.

From the prettier docs:

    At first glance, avoiding parentheses may look like a better choice because of less visual noise. However, when 
    Prettier removes parentheses, it becomes harder to add type annotations, extra arguments or default values as well  
    as making other changes. Consistent use of parentheses provides a better developer experience when editing real 
    codebases, which justifies the default value for the option.

I tend to agree with their rational